com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/Logic/OrNode.cs

```diff
@@ -9,12 +8,14 @@ internal class OrNode : IStandardNode
         public static int Version = 1;
```

Comment:
I think all implementers of `IStandardNode.Version` need to use the `=>` to get caught by reflection.

Reply:
Oh wait-- we opted to let the FunctionDescriptor have a version- so we're not doing that anymore-- okay nevermind!

Reply:
You're right. I need to change these.

Reply:
Changed